src/fosslight_util/output_format.py

```diff
@@ -9,20 +9,25 @@
 SUPPORT_FORMAT = {'excel': '.xlsx', 'csv': '.csv', 'opossum': '.json'}
 
 
-def check_output_format(output='', format=''):
+def check_output_format(output='', format='', customized_format={}):
     success = True
     msg = ''
     output_path = ''
     output_file = ''
     output_extension = ''
 
+    if customized_format != {}:
```

Comment:
How about changing to `if customized_format: `?

Reply:
it's bettet than mine.
I'll modify it